### ISSUE TITLE
Have you trained sphereface 64 when m=0.35 and s=64, just in the condition of the paper? I 

### ISSUE THREAD

**Issue by Reneea1** [MuggleWang/CosFace_pytorch#5]
I trained sphereface64 network in the same super-parameter setting, but can not reach 99.% accuracy in LFW, but sphereface20 network can get 99.20% accuracy in LFW, which make me confused...why the deeper network works worse than the shallow network?

**Comment by weihua04** [comment]
which dataset do u use? may be training set is not large enough for the deeper network
